WcaOnRails/lib/solve_time.rb

```diff
@@ -166,21 +166,21 @@ def clock_format
           result = "0:#{time_seconds}"
         else
           while time_seconds >= 60
-            result = ":%02d#{result}" % ( time_seconds % 60 )
-            time_seconds = time_seconds / 60
+            result = format(":%02d#{result}", time_seconds % 60)
```

Comment:
I'm also used to the `%` syntax, courtesy of python. However, I do think that it's a weird use of the character, so I like this rule.